owl/SOMA-HOME.owl

```diff
@@ -106,6 +130,30 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-HOME.owl#ClosingDisposition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-HOME.owl#ClosingDisposition">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Linkage"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isQualityOf"/>
+                <owl:allValuesFrom>
+                    <owl:Restriction>
+                        <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasQuality"/>
+                        <owl:someValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Containment"/>
+                    </owl:Restriction>
+                </owl:allValuesFrom>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>The disposition of an object to connect with some other, so that a region of space becomes fully enclosed by the object and separate from the exterior. 
+
+Often this connection is between parts of the same object.
+
+Because of the existence of a separation between some interior and exterior space regions, objects that can close are also objects that can contain.</rdfs:comment>
+    </owl:Class>
```

Comment:
Two questions: 
1. Why do we use isQualityOf and hasQuality instead of isDispositionOf and hasDisposition?
2. ClosingDisposition -> Containment might be valid, however, does then a room have the disposition of being closeable or the door? Because, I guess, a door does not have the containment disposition, but a room does?